### PR TITLE
a New branch

### PR DESCRIPTION
**Pull request by TebohoRomeo**

### CODE REVIEW

**Comment by TebohoRomeo** [comment]
A covid 19 information pulled

**Review by Dibwe243 — CHANGES_REQUESTED** [review]
Hi Romeo,

Please create a proper Readme file. Could you also please create a pull request for the entire project I need to add my comment to it.

Thank you

File: README.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-
 ## Available Scripts
 
 In the project directory, you can run:
```
Comment (leanny01) [inline README.md:3]: You have done well to add a description, there are few things missing though,  please see bellow suggestions
 It is recommended to guide the user step by step on how to run the app
```suggestion

1. move to the project directory within your terminal `cd  ~/Projecte_directory`
2. install dependencies ` npm run install`
3. start the app `npm run start `
```

Please be mindful of styling, feel free to google the internet for ideas.

Comment (leanny01) [reply README.md:3]: 1. move to the project directory within your terminal `cd  ~/Projecte_directory`
2. install dependencies ` npm run install`
3. start the app `npm run start `

File: README.md [hunk 2]
```diff
@@ -1,5 +1,3 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-
 ## Available Scripts
```
Comment (leanny01) [inline README.md:1]: This could be 
### Instalation instructions

File: README.md [hunk 3]
```diff
@@ -9,62 +7,10 @@ In the project directory, you can run:
...
-### Deployment
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
-
-### `npm run build` fails to minify
+This is a React app that pulls covid-19 information from the https://covid19api.com/ API Website
 
-This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
-covid-19
-# Covid-19
```
Comment (leanny01) [inline README.md:70]: Please ring back this tile 

#### Covid-19

**Review by leanny01 — CHANGES_REQUESTED** [review]

**Review by leanny01 — APPROVED** [review]
Well done!

**Pull request merged**